frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx

```diff
@@ -76,13 +76,13 @@ export const DWWorkloadsTable: React.FC = () => {
 
   return (
     <Card>
-      <CardTitle>Workloads</CardTitle>
+      <CardTitle>Distributed Workloads</CardTitle>
       <CardBody>
         <Table
           enablePagination
           data={workloads.data}
           columns={columns}
-          emptyTableView={<>No workloads match your filters</>}
+          emptyTableView={<>No distributed workload metrics match your filters</>}
```

Comment:
Well, in this specific case this is a table of Distributed Workloads, not metrics, so I think it should be "No distributed workloads match your filters". @xianli123 does that make sense in the context of the empty state here inside the Distributed Workloads table? "Distributed Workload Metrics" makes sense for the page title and top-level descriptions/errors, but here in this table the workloads themselves are the objects being listed/filtered.

Reply:
Is this for the "no results" state? If so, I agree with @mturley 's opinion. "No distributed workloads match your filters" is better. cc @gitdallas

Reply:
That's correct